packages/web/tests/components/UserProfile/EditUserForm.test.tsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import { render, screen, waitFor } from '../../testUtils/testTools';
+import userEvent from '@testing-library/user-event';
+import { EditUserForm, EditUserProps } from '../../../src/components/userprofile/EditUserForm';
+import fetchMock from 'fetch-mock-jest';
+import { User } from '../../../src/pages/user/[uid]';
+
+const mockEditFormProps: EditUserProps = {
+  setEditToggle: jest.fn(React.useState).mockImplementation(),
+  setUser: jest.fn(React.useState).mockImplementation(),
+  user: {
+    id: '123',
+    name: 'Bill Sigh',
+    pronouns: 'he/him',
+    schoolName: 'School of Bill',
+  },
+};
+
+const mockReturnedUser = {
+  id: '123',
+  name: 'Bill High',
+  pronouns: 'he/him',
+  schoolName: 'School of Hill',
+};
+
+describe('', () => {
+  beforeEach(async () => {
+    jest.clearAllMocks();
+    fetchMock.reset();
+  });
+
+  it('Error message when try to enter no name', async () => {
```

Comment:
`it` verbiage should read like a sentence, so capitalization and grammar is fairly important. The original text results in this sentence:
> _it Error message when try to enter no name_

See the problem? 
```suggestion
  it('displays an error when the user tries to submit without a name', async () => {
```